frontend/talentsearch/src/js/components/Home/partials/Hero/Hero.tsx

```diff
@@ -0,0 +1,109 @@
+import React from "react";
+import { useIntl } from "react-intl";
+
+import Heading from "@common/components/Heading";
+import { imageUrl } from "@common/helpers/router";
+
+import CallToAction from "../CallToAction";
+
+import { useTalentSearchRoutes } from "../../../../talentSearchRoutes";
+import { useDirectIntakeRoutes } from "../../../../directIntakeRoutes";
+import TALENTSEARCH_APP_DIR from "../../../../talentSearchConstants";
+
+import "./hero.css";
+
+const landscapeRandomize = () => {
+  const items = [
+    imageUrl(TALENTSEARCH_APP_DIR, "hero-1-landscape.jpg"),
+    imageUrl(TALENTSEARCH_APP_DIR, "hero-2-landscape.jpg"),
+    imageUrl(TALENTSEARCH_APP_DIR, "hero-3-landscape.jpg"),
+    imageUrl(TALENTSEARCH_APP_DIR, "hero-4-landscape.jpg"),
+  ];
+  return items[Math.floor(Math.random() * items.length)];
```

Comment:
Is a fully randomized picker going to trigger Chromatic diffs?

Reply:
I believe this should be resolved with [stabilize random header image for chromatic](https://github.com/GCTC-NTGC/gc-digital-talent/pull/4049/commits/8434ac49ae065d3862266fe34b71464044c2678c).

Kind of wish there was a better way of doing this 🤔